src/dsa/rpo_falcon512/math/mod.rs

```diff
@@ -85,6 +91,15 @@ pub(crate) fn ntru_gen<R: Rng>(n: usize, rng: &mut R) -> [Polynomial<i16>; 4] {
     loop {
```

Comment:
Should we put some kind of a "break" on this loop? That is, if run it a lot of times and don't get a result, maybe we should error out (similar to what we do in another loop).

Reply:
I think that this loop is guaranteed to break. In addition, supposing that we put a break after iterating a max number of iterations, then the user will just have to re-run the key generation procedure again but this is not different from letting the original run for twice the number of iterations.
So I am on the fence on this from a practical point of view but from a security point of view it doesn't hurt as far as I can see.

Reply:
I think it is fine to keep as is then.